internal/venice-common/src/main/java/com/linkedin/venice/meta/ServerAdminAction.java

```diff
@@ -1,8 +1,19 @@
 package com.linkedin.venice.meta;
 
+import java.util.HashMap;
+import java.util.Map;
+
+
 public enum ServerAdminAction {
   DUMP_INGESTION_STATE(0), DUMP_SERVER_CONFIGS(1);
 
+  private static final Map<Integer, ServerAdminAction> ADMIN_ACTION_MAP = new HashMap<>(2);
+
+  static {
+    for (ServerAdminAction action: values()) {
+      ADMIN_ACTION_MAP.put(action.getValue(), action);
+    }
+  }
```

Comment:
FYI, for the common case of enum codes that range from 0..N monotonically, we have a cleaner solution than using a map... See `EnumUtils::getEnumValuesArray` and `VeniceEnumValue`. It is very easy to use, more performant, and will require a bit fewer lines of code than you have here...